### DISCUSSION TITLE
Color Contents, List of figures; List of Tables in Black

### DISCUSSION THREAD

**Discussion by 028ara** [joaomlourenco/novathesis#501] [Q&A] [answered]
I must put all the text in black for the University of Minho. This is a rule.

<img width="554" alt="Captura de ecrã 2024-05-06, às 14 16 37" src="https://github.com/joaomlourenco/novathesis/assets/161602875/8aa20899-3bef-4a19-84b2-757bc5376df3">

**Comment by joaomlourenco** [comment, marked as answer]
In `Config/1_novathesis.tex`, uncomment the option and set the color to black:
```latex
\ntsetup{color/links=black}
```

**Reply by 028ara** [reply]
Obrigada :)
